src/olympia/landfill/collection.py

```diff
@@ -33,18 +34,17 @@ def create_collection(application):
     return c
 
 
-def generate_collection(addon, app=None):
+def generate_collection(addon, app=None, **kwargs):
```

Comment:
yep, makes sense.  It was https://github.com/mozilla/addons-server/pull/5053/files#diff-2107fc25c4e6b55be933c6498a51fba8R122 that didn't.